packages/ui/__tests__/SelectField.test.tsx

```diff
@@ -210,4 +210,45 @@ describe('SelectField', () => {
 
     expect(wrapper.find(IconButton).prop('icon')).toBe(X)
   })
+
+  it('Renders multiple selections with correct props', async () => {
+    const onChange = jest.fn()
+
+    const value = [selectOptions[0], selectOptions[1]]
+    const wrapper = await mountAndCheckA11Y(
+      <SelectField name={selectName} label={selectLabel} isMulti={true} options={selectOptions} value={value} onChange={onChange} />,
+      {
+        // xxx axeOptions: SELECT_FIELD_AXE_OPTIONS,
+      },
```

Comment:
Do we need this?

Reply:
it seems it's gone from the master too, removing...